PhysicsTools/NanoAOD/python/nanoDQM_cff.py

```diff
@@ -18,6 +18,106 @@
     plots = _boostedTauPlotsV10
 )
 
+## EGamma custom nano
+_Electron_extra_plots = cms.VPSet()
```

Comment:
a more elegant way would be to do  `_Electron_extra_plots = nanoDQM.vplots.Electron.plots.copy()` IMO.
I'd appreciate if you could make this change (independent of the purpose of this PR) every in this file where there is a full copy of a VPSet